static/js/zamboni/reviewers.js

```diff
@@ -86,14 +86,14 @@ $(document).ready(function () {
 
     $('#addon-queue-filter-form .select-all').click(function (e) {
       e.preventDefault();
-      jQuery('#addon-queue-filter-form input[type="checkbox"]').prop(
+      $('#addon-queue-filter-form input[type="checkbox"]').prop(
```

Comment:
It just occurred to me.. What does _that_ fix?

Reply:
It fixes compliance with the eslint rule that we should not reference undefined globals in our own code. We do define both `$` and `jQuery` because we have vendored code and node module code that relies on it. But we can hold ourselves to a stricter standard and `$` was already imported in the module scope.